web/modules/custom/dpl_event/dpl_event.install

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * DPL Event install file. These hooks will be run *before* config import.
+ */
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Delete the deprecated event description fields, prior to CIM.
+ *
+ * This field has been replaced by field_description.
+ * The data has already been migrated, in a prior deploy.
+ */
+function dpl_event_update_10002(): string {
+  $types = ['eventseries', 'eventinstance'];
+  $message = '';
+
+  foreach ($types as $type) {
+    $storage = FieldStorageConfig::loadByName($type, 'field_event_description');
+
+    if ($storage instanceof FieldStorageConfig) {
+      $storage->delete();
+
+      $message .= t('All field_event_description storage of @type has been deleted.', ['@type' => $type], ['context' => 'DPL admin UX'])->render();
+    }
+
+    $config = FieldConfig::loadByName($type, 'default', 'field_event_description');
+
+    if ($config instanceof FieldConfig) {
+      $config->delete();
+      $message .= t('All field_event_description config of @type has been deleted.', ['@type' => $type], ['context' => 'DPL admin UX'])->render();
+    }
+  }
```

Comment:
I wonder if there a reason we do not do the deletion of the field content as a part of #843.

I have not does this type of work very much.

Reply:
I could also have done that, but, I wanted to make sure that the data was migrated first, and then the old field deleted in this PR.
No matter what, we needed to do two deploys regardless, as I also had to export the config.

Reply:
:+1: You are right. It makes sense to keep the old data a bit in case of errors.